src/Aks/Aks/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Fixed the issue that `Set-AzAksCluster` can't upgrade cluster in some cases. [#14583]
```

Comment:
do we know what case is?

Reply:
When the version of node pool doesn't match cluster's version.